chargeOclock/Source/Other/NetworkAPI.swift

```diff
@@ -55,6 +58,29 @@ class NetworkAPI {
 			}.resume()
 	}
 
+	fileprivate func performDeleteRequest(to uri: String, completion: @escaping NetworkCompletion) {
```

Comment:
I have no real problems with this method, but you could go further 😁 

create a (private? 🤔) method `performRequest` that accepts the http method to perform. Then call this method from both `performGetRequest` and `performDeleteRequest`.

later, when we add `performPostRequest` (and others), we'll reuse the `performRequest` method and just add parameters as they will be needed.